data/level_to_cpm.json

```diff
@@ -0,0 +1,81 @@
+{
+  "1":    0.094,
```

Comment:
Then we need to do some test whether these values still accurate or not. 
Niantic (or pgoapi) added cp_multiplier on Jul 29, and additional_cp_multiplier on Aug 9. 
If it really from Niantic. I believe they've nerf/buff some Pokemons and these static CPM gonna be useless.